hedera-node/src/main/java/com/hedera/services/stream/RecordStreamFileWriter.java

```diff
@@ -631,16 +637,19 @@ private SignatureObject generateSignatureObject(final byte[] hash) {
     private void createSidecarFile(final Builder sidecarFileBuilder, final File sidecarFile)
             throws IOException {
         try (FileOutputStream stream = new FileOutputStream(sidecarFile, false);
+                GZIPOutputStream gzipStream = new GZIPOutputStream(stream);
                 SerializableDataOutputStream dos =
                         new SerializableDataOutputStream(
                                 new BufferedOutputStream(
-                                        new HashingOutputStream(sidecarStreamDigest, stream)))) {
+                                        new HashingOutputStream(
+                                                sidecarStreamDigest, gzipStream)))) {
```

Comment:
This is a breaking change. The hashing should continue to be on the uncompressed bytes.

Reply:
Nevermind, I can see now it's still hashing the uncompressed bytes.